nextflow_schema.json

```diff
@@ -237,7 +237,7 @@
         },
         "master_template": {
             "type": "string",
-            "default": "assets/master_template.html"
+            "default": "/home/proj/development/microbial/metagenomics/sofia_workspace/gms_16S/assets/master_template.html"
```

Comment:
Same here:Please remove user-specific part in the path.

Reply:
Fixed now.